.github/workflows/tests.yaml

```diff
@@ -68,11 +65,5 @@ jobs:
                 php bin/console sass:build
                 php bin/console asset-map:compile
 
-            - name: "Install PHPUnit"
-              run: vendor/bin/simple-phpunit install
-
-            - name: "PHPUnit version"
-              run: vendor/bin/simple-phpunit --version
-
             - name: "Run tests"
-              run: vendor/bin/simple-phpunit ${{ env.PHPUNIT_FLAGS }}
+              run: bin/phpunit
```

Comment:
shouldn't this be `vendor/bin/phpunit` ?

Reply:
The phpunit/phpunit recipe also installs `bin/phpunit`. This is done to make migrating away from our simple-phpunit wrapper easy and without impacting documentation (as both use the same helper script).

I'm not sure why this project used the vendor binary in the CI (as the `bin/phpunit` file is mentioned in the README).